quickwit-cli/src/main.rs

```diff
@@ -25,9 +25,9 @@ use opentelemetry::global;
 use opentelemetry::sdk::propagation::TraceContextPropagator;
 use quickwit_cli::cli::{build_cli, CliCommand};
 use quickwit_cli::QW_JAEGER_ENABLED_ENV_KEY;
-use quickwit_cluster::QuickwitService;
 use quickwit_common::metrics::new_gauge;
 use quickwit_common::runtimes::RuntimesConfiguration;
+use quickwit_common::service::QuickwitService;
```

Comment:
I want to keep `quickwit-common` free of Quickwit-specific business logic. So let's move this code to `quickwit-config` instead.